### PR TITLE
more tutorial tweaks

### PR DESCRIPTION
**Pull request by drammock**
moderate overhaul of freesurfer background tutorial; the rest is just cleanup of some old TODO items, and removal of no-longer-required `:class:` tags along the way.

### CODE REVIEW

**Comment by drammock** [comment]
This is ready for review/merge.

File: tutorials/intro/20_events_from_raw.py [hunk 1]
```diff
@@ -108,12 +107,12 @@
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
 # If your data has events recorded on a STIM channel, you can convert them into
-# an events array using :func:`mne.find_events`. The sample number of the onset
+# an events array using `mne.find_events`. The sample number of the onset
```
Comment (agramfort) [inline tutorials/intro/20_events_from_raw.py:110]:
```suggestion
# an events array using `~mne.find_events`. The sample number of the onset
```

File: tutorials/intro/20_events_from_raw.py [hunk 2]
```diff
@@ -127,9 +126,9 @@
 #     code was on the immediately preceding sample. In practice, that value is
 #     almost always ``0``, but it can be used to detect the *endpoint* of an
 #     event whose duration is longer than one sample. See the documentation of
-#     :func:`mne.find_events` for more details.
+#     `mne.find_events` for more details.
```
Comment (agramfort) [inline tutorials/intro/20_events_from_raw.py:129]:
```suggestion
#     `~mne.find_events` for more details.
```

File: tutorials/intro/20_events_from_raw.py [hunk 3]
```diff
@@ -139,12 +138,12 @@
...
 # a time-saver to configure that variable once and then forget about it.
 #
-# :func:`~mne.find_events` has several options, including options for aligning
+# `~mne.find_events` has several options, including options for aligning
 # events to the onset or offset of the STIM channel pulses, setting the minimum
 # pulse duration, and handling of consecutive pulses (with no return to zero
 # between them). For example, you can effectively encode event duration by
-# passing ``output='step'`` to :func:`mne.find_events`; see the documentation
-# of :func:`~mne.find_events` for details. More information on working with
+# passing ``output='step'`` to `mne.find_events`; see the documentation
```
Comment (agramfort) [inline tutorials/intro/20_events_from_raw.py:145]:
```suggestion
# passing ``output='step'`` to `~mne.find_events`; see the documentation
```

File: tutorials/intro/20_events_from_raw.py [hunk 4]
```diff
@@ -287,7 +285,7 @@
 
 ###############################################################################
 # Other examples of resting-state analysis can be found in the online
-# documentation for :func:`mne.make_fixed_length_events`, such as
+# documentation for `mne.make_fixed_length_events`, such as
```
Comment (agramfort) [inline tutorials/intro/20_events_from_raw.py:288]:
```suggestion
# documentation for `~mne.make_fixed_length_events`, such as
```

**Review by agramfort — APPROVED** [review]
Besides looks 👍

**Review by larsoner — APPROVED** [review]

**Pull request merged**